internal/processing/processors/v2alpha1/reconciliation.go

```diff
@@ -40,8 +40,8 @@ func (r Reconciliation) GetProcessors() []processing.ReconciliationProcessor {
 	return r.processors
 }
 
-func NewReconciliation(apiRuleV2alpha1 *gatewayv2alpha1.APIRule, apiRuleV1beta1 *gatewayv1beta1.APIRule, validator validation.ApiRuleValidator, config processing.ReconciliationConfig, log *logr.Logger) Reconciliation {
-	vsProcessor := istio.Newv1beta1VirtualServiceProcessor(config, apiRuleV1beta1)
+func NewReconciliation(apiRuleV2alpha1 *gatewayv2alpha1.APIRule, apiRuleV1beta1 *gatewayv1beta1.APIRule, validator validation.ApiRuleValidator,config processing.ReconciliationConfig, log *logr.Logger) Reconciliation {
```

Comment:
I wonder how your linter didn't pick It up.
```suggestion
func NewReconciliation(apiRuleV2alpha1 *gatewayv2alpha1.APIRule, apiRuleV1beta1 *gatewayv1beta1.APIRule, validator validation.ApiRuleValidator, config processing.ReconciliationConfig, log *logr.Logger) Reconciliation {
```

Reply:
I fixed the merge conflict in the GitHub UI, so this is at least why it wasn't fixed with auto format 🙈